akka-actor-typed/src/main/mima-filters/2.10.0.backwards.excludes/32499-simplify-flight-recorder.excludes

```diff
@@ -0,0 +1,8 @@
+# #32499 chore: Simplify flight recorder interactions
```

Comment:
The file name should be the latest release. Did this solve anything? Is it needed?

Reply:
I changed the name 👍  

> Did this solve anything? Is it needed?

When I remove the content of the file, I get this error:
```
sbt:akka-actor> mimaReportBinaryIssues
[error] akka-actor: Failed binary compatibility check against com.typesafe.akka:akka-actor_2.13:2.6.7! Found 2 potential problems (filtered 112)
[error]  * class akka.util.FlightRecorderLoader does not have a correspondent in current version
[error]    filter with: ProblemFilters.exclude[MissingClassProblem]("akka.util.FlightRecorderLoader")
[error]  * object akka.util.FlightRecorderLoader does not have a correspondent in current version
[error]    filter with: ProblemFilters.exclude[MissingClassProblem]("akka.util.FlightRecorderLoader$")
[error] stack trace is suppressed; run last mimaReportBinaryIssues for the full output
[error] (mimaReportBinaryIssues) Failed binary compatibility check against com.typesafe.akka:akka-actor_2.13:2.6.7! Found 2 potential problems (filtered 112)
[error] Total time: 0 s, completed Sep 4, 2024, 9:44:43 AM
sbt:akka-actor>
```